wolfcrypt/src/aes.c

```diff
@@ -12975,6 +12976,29 @@ int wc_AesXtsEncryptUpdate(XtsAes* xaes, byte* out, const byte* in, word32 sz,
     return ret;
 }
 
+int wc_AesXtsEncryptUpdate(XtsAes* xaes, byte* out, const byte* in, word32 sz,
+                           byte *i)
+{
+    if (sz & ((word32)AES_BLOCK_SIZE - 1U))
+        return BAD_FUNC_ARG;
+    return AesXtsEncryptUpdate(xaes, out, in, sz, i);
+}
+
+int wc_AesXtsEncryptFinal(XtsAes* xaes, byte* out, const byte* in, word32 sz,
+                           byte *i)
+{
+    int ret;
+    if (sz > 0)
+        ret = AesXtsEncryptUpdate(xaes, out, in, sz, i);
+    else
+        ret = 0;
```

Comment:
I don't understand the point of final, since it cannot return anything to "out".